hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterMergeTask.java

```diff
@@ -106,19 +100,6 @@ private void disposeTasks(Collection<Runnable>... tasks) {
         }
     }
 
-    private void tryToPromoteLocalLiteMember() {
-        if (wasLiteMember) {
```

Comment:
`wasLiteMember` field can be deleted too.

Reply:
done